src/main/scala/djinni/YamlGenerator.scala

```diff
@@ -154,6 +154,7 @@ class YamlGenerator(spec: Spec) extends Generator(spec) {
     "header" -> QuotedString(objcMarshal.include(td.ident)),
     "boxed" -> QuotedString(objcMarshal.boxedTypename(td)),
     "pointer" -> objcMarshal.isPointer(td),
+    "generic" -> false,
```

Comment:
Is there a reason why the Objective-C part has a default value for the key "generic" defined here, while the C++/CLI part doesn't (`private def cs(td: TypeDecl)`)?

Reply:
No, the change was made such that the parameter is optional. The Java code is doing the same thing as here, so I mimicked it in the objc, just forgot to do the same with the CLI part.

Reply:
Can you please give me an update on this? 
Because from this conversation it is not totally clear to me if we shall wait with merging this PR until we have one more change, or if it is already in it's final state and the thing you discuss here is not important.

Reply:
I pushed the corresponding change for c++/cli

Reply:
Ok, thanks for the info! I guess that means it's ok to merge.